src/styles/vars.css

```diff
@@ -37,11 +37,16 @@
   --btn-shadow-color: var(--gray);
   --btn-shadow-dark-color: var(--dark-gray);
   --btn-shadow-darker-color: var(--dark-slate-gray);
-  --axis-primary-color: var(--dark-slate-gray);
-  --axis-secondary-color: var(--gray);
 }
 
 :root {
   --h5w-line--curveColor: var(--secondary-dark);
   --h5w-line--auxColor: var(--secondary);
+  --h5w-matrix--fontFamily: var(--monospace);
+  --h5w-matrix--cellBgAlt: var(--secondary-light-bg);
+  --h5w-matrix--indexCellBg: var(--primary-bg);
+  --h5w-matrix--indexCellBgAlt: var(--primary-light-bg);
+  --h5w-matrix--anchorCellBg: var(--primary-dark-bg);
+  --h5w-scalar-fontFamily: var(--monospace);
+  --h5w-raw-fontFamily: var(--monospace);
 }
```

Comment:
To see what it looks like with the default values, you can pull the branch and comment out this block.